MalApi/Anime/UpdateAnimeObject.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.Xml.Serialization;
+
+namespace MalApi
+{
+    [XmlRoot("entry")]
+    public class UpdateAnimeObject : UpdateObjectBase
```

Comment:
"Object" is kind of generic. "AnimeUpdate" would describe the purpose as well.

Reply:
The properties in this class are all strings. The users of the library should be dealing with integers for episode count, AnimeCompletionStatus for status, etc, instead of expecting them to serialize those properties to strings.

Reply:
What does it mean if a property is not set? Does it not get updated or does it get set to an empty value in MAL? This should be documented in a doc comment on the class.

Reply:
Might be nice to link to the MAL documentation at https://myanimelist.net/modules.php?go=api#animevalues in a comment.

Reply:
I agree with all your comments.